fineract-provider/build.gradle

```diff
@@ -57,6 +57,7 @@ buildscript {
         classpath "com.diffplug.spotless:spotless-plugin-gradle:5.1.1"
         classpath "io.swagger.core.v3:swagger-gradle-plugin:2.1.4"
         classpath "gradle.plugin.org.hidetake:gradle-swagger-generator-plugin:2.18.2"
+        classpath "org.springframework.security.oauth:spring-security-oauth2:2.3.3.RELEASE"
```

Comment:
please remove, this is wrong and not required, as `spring-security-oauth2` is obviously a dependency at runtime (below) and not a `buildscript` dependency for Gradle itself (which is what this section is for).